test/integration/psetv2.spec.ts

```diff
@@ -166,9 +185,9 @@ describe('liquidjs-lib (transactions with psetv2)', () => {
     //  - 0 amount is used to set the pset output's script to OP_RETURN.
     //
     const outputs = [
-      new CreatorOutput(lbtc, 60000000, bob.payment.address!, 0),
-      new CreatorOutput(lbtc, 39999500, alice.payment.address!, 0),
-      new CreatorOutput(lbtc, 0, alice.payment.confidentialAddress!, 0),
+      new CreatorOutput(lbtc, 60000000, bob.payment.output),
+      new CreatorOutput(lbtc, 39999500, alice.payment.output),
+      new CreatorOutput(lbtc, 0, Buffer.of(OPS.OP_RETURN), randomBytes(33), 0),
```

Comment:
`randomBytes` cannot be used for public keys unless we manually put a 02 or 03 prrefix in front of that, better hardcoding one or generating a real random one

Reply:
that was the problem 🙏